README.md

```diff
@@ -245,6 +245,37 @@ The command below can be used to run the glibc tests:
 
     make check-glibc-linux
 
+### LLVM / clang
+
+LLVM can be used in combination with the RISC-V GNU Compiler Toolchain
+to build RISC-V applications. To build LLVM with C and C++ support the
+configure flag `--enable-llvm` can be used.
+
+E.g. to build LLVM on top of a RV64 Linux toolchain the following commands
+can be used:
+
+  ./configure --prefix=$RISCV --enable-llvm --enable-linux
+  make
+
+Note, that a combination of `--enable-llvm` and multilib configuration flags
+is not supported.
+Also note, that building LLVM is only supported in combination with building
```

Comment:
Just curious, what is the reason for restricting support to the RISC-V Linux toolchain only and not also the bare metal/newlib/embedded toolchain? Is it a question of a lack of resources to extend support to the latter or is it due to some inherent technical restriction? Thanks.

Reply:
I guess one reason is multi-lib support is kind of...limited on LLVM site, I've a patch to fix that, but apparently I don't found enough time to complete that on upstream: https://reviews.llvm.org/D97916

Reply:
Maybe we could avoid the multi-lib path for LLVM with newlib support.